services/tenant-ui/README.md

```diff
@@ -90,3 +90,15 @@ The Tenant UI Inkeeper functionality can be configured to log in with either (or
 To set up the OIDC provider of your choice, add configuration values in your deployment to match the `frontend.innkeeperOidc` fields for a auth code grant client,a nd configure the `server.oidc` fields appropriately to veify the JWKS for tokens from that client. 
 
 As well, the Innkeeper secret must be available to the Tenant UI server, this is set in `server.innkeeper` configuration...
+
+## Configuring Matomo
+
+If you would like to use Matomo for tracking you can set the `FRONTEND_MATOMO_URL`
+environment variable as a one time override to the url exposed by your Matomo instance.
+If you would like a more permanent solution set `frontend.matomoUrl` in
+[custom-environment-variables.json](../config/custom-environment-variables.json)
```

Comment:
Wouldn't say it's a one-time solution or permanent solution (in fact, no values should be explicitly set in custom-environment-variables, that file is for pulling environment variables into node config)

Would just leave it as "If you would like to use Matomo for tracking you can set the `FRONTEND_MATOMO_URL` environment variable as exposed in [custom-environment-variables.json](../config/custom-environment-variables.json)"